packages/bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -144,36 +146,45 @@ class BpkBannerAlert extends Component {
     */
     // Disabling 'click-events-have-key-events and interactive-supports-focus' because header element is not focusable.
     // ToggleButton is focusable and works for this.
-    return !show ? null : (
-      <section className={sectionClassNames.join(' ')} {...rest}>
-        <header
-          role={ariaRoles.join(' ')}
-          aria-live={ariaLive}
-          className={headerClassNames.join(' ')}
-          onClick={this.onExpand}
-        >
-          <span className={getClassName('bpk-banner-alert__icon')}>{getIconForType(type)}</span>
+    return (
+      <AnimateAndFade
+        className={className}
+        animateOnEnter={animateOnEnter}
+        animateOnLeave={dismissable || animateOnLeave}
+        show={show}
+      >
+        <section className={sectionClassNames.join(' ')} {...rest}>
+          <header
+            role={ariaRoles.join(' ')}
+            aria-live={ariaLive}
+            className={headerClassNames.join(' ')}
+            onClick={this.onExpand}
+          >
+            <span className={getClassName('bpk-banner-alert__icon')}>{getIconForType(type)}</span>
           &nbsp;
-          <span className={getClassName('bpk-banner-alert__message')}>{message}</span>
+            <span className={getClassName('bpk-banner-alert__message')}>{message}</span>
           &nbsp;
-          {isExpandable ? (
-            <span className={getClassName('bpk-banner-alert__toggle')}>
-              <ToggleButton expanded={isExpanded} label={toggleButtonLabel} />
-            </span>
+            {isExpandable ? (
+              <span className={getClassName('bpk-banner-alert__toggle')}>
+                <ToggleButton expanded={isExpanded} label={toggleButtonLabel} />
+              </span>
           ) : null}
```

Comment:
You can simplify this logic:

`isExpandable ? <span ...> : null` can be written as `isExpandable && <span ...>`.

Same for the `dismissable` bit further down. It'll just save you from having to add the `null` part.